lib/project_types/script/messages/messages.rb

```diff
@@ -64,10 +64,18 @@ module Messages
                                               "%{missing_keys}.",
           config_ui_missing_keys_error_help: "Add the keys and try again.",
 
+          config_ui_invalid_type_error_cause: "The UI configuration file %{filename} only accept "\
+                                              "one of the following type(s): %{valid_types}.",
+          config_ui_invalid_type_error_help: "Change the types and try again.",
+
           config_ui_fields_missing_keys_error_cause: "A field entry in the UI configuration file %{filename} is "\
                                                      "missing required keys: %{missing_keys}.",
           config_ui_fields_missing_keys_error_help: "Add the keys and try again.",
 
+          config_ui_fields_missing_keys_error_cause: "The UI configuration file %{filename} fields only accept "\
+                                                     "one of the following type(s): %{valid_types}.",
+          config_ui_fields_missing_keys_error_help: "Change the types and try again.",
```

Comment:
I think these keys need to be renamed
```suggestion
          config_ui_fields_invalid_type_error_cause: "The UI configuration file %{filename} fields only accept "\
                                                     "one of the following type(s): %{valid_types}.",
          config_ui_fields_invalid_type_error_help: "Change the types and try again.",
```